### DISCUSSION TITLE
AI-Based Animation of Generated Images

### DISCUSSION THREAD

**Discussion by tim-gromeyer** [lllyasviel/Fooocus#1560] [Q&A]
Greetings,

I’m interested in creating animations from generated images using AI. Specifically, I’d like to animate a scene, such as a living room with a fireplace, and snow falling outside the window. I’m looking for a solution that doesn’t involve traditional software like Photoshop or CapCut. Any guidance on how to achieve this would be greatly appreciated.

Thank you!

**Comment by crazyanageek** [comment]
try this :
https://github.com/guoyww/animatediff/
